express-notes-api/server.ts

```diff
@@ -0,0 +1,111 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+
+const app = express();
+app.use(express.json());
+
+type Data = {
+  nextId: number;
+  notes: Record<string, string>;
```

Comment:
This isn't the correct type. Look in data.json to see what the type should look like.

It should be `notes: Record<number, Note>;` where Note has `id` and `content` fields.